rmw_connext_cpp/CMakeLists.txt

```diff
@@ -17,17 +17,19 @@ if(NOT Connext_FOUND)
 endif()
 
 find_package(rmw REQUIRED)
+find_package(rmw_connext_shared_cpp REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
 find_package(rosidl_typesupport_connext_cpp REQUIRED)
 
-ament_export_definitions(${Connext_DEFINITIONS})
+ament_export_definitions(${rmw_connext_shared_cpp_DEFINITIONS})
 ament_export_dependencies(rmw rosidl_generator_cpp rosidl_typesupport_connext_cpp)
-ament_export_include_directories("${Connext_INCLUDE_DIRS}")
+ament_export_include_directories("${rmw_connext_shared_cpp_INCLUDE_DIRS}")
 
-link_directories(${Connext_LIBRARY_DIRS})
+link_directories(${rmw_connext_shared_cpp_LIBRARIES})
```

Comment:
This has been addressed in https://github.com/ros2/rmw_connext/commit/5eb0a187bd865936049dfead1bc6ee4fe22d0392